hackathon_site/registration/test_forms.py

```diff
@@ -138,6 +151,16 @@ def test_fields_are_required(self):
         self.assertIn("resume", form.errors)
         self.assertIn("This field is required.", form.errors["resume"])
 
+    def test_with_optional_fields(self):
+        data = self.data.copy()
+        data["address1"] = "35 St.George St"
+        data["address2"] = "Apt. No. 13"
+        data["city"] = "Toronto"
+        data["region"] = "Ontario"
+
+        form = self._build_form(data=data)
+        self.assertTrue(form.is_valid())
+
```

Comment:
Is only the "Country" field required? In that case, the "Address Information" header should be changed to "Address Information (Country required, all else optional)" or like a subtitle to address this. Are we shipping them swag? Maybe have a subtitle like "If you want swag, please include your address?"

Anyone have a better way of phrasing this? I want people to recognize that they don't have to give their address unless they want to, but we need their country regardless

Reply:
Actually, I think I misunderstood logistics, they want the full address of people unless they live outside of Canada, so in that case I'll make all the fields required. But yeah, I'll confirm what they need the address for